uiowa/uiowa
# Implement google search schema for uiowa.edu frontpage

Resolves: #1326 

## Test
- Successfully pull and apply schema_metatag patch.
- Import uiowa.edu specific front page metatag configuration.
- See the added schema info in the `<script type="application/ld+json">` tag that matches what is in the issue text file #1326
